cooked-validators.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 3.4
 
--- This file has been generated from package.yaml by hpack version 0.36.1.
+-- This file has been generated from package.yaml by hpack version 0.36.0.
```

Comment:
That's weird. Have we rolled back our hpack?

Reply:
yes that's super weird indeed let me look into it

Reply:
ok it's no longer present